supplementary/devices/nicki.rst

```diff
@@ -62,9 +62,11 @@ Special boot modes
 
 Fastboot: With the device powered off, while holding Volume Up, connect the USB cable to the computer. The LED should turn blue.
 
-Recovery: On boot, press Volume Down when the LEDs start lighting up. This only works with a working boot image. If there is none, you can flash twrp to boot partition first and use it to reboot into recovery.
+Flashmode: With the device powered off, while holding Volume Down, connect the USB cable to the computer. The LED should turn green. Then you can go back to the stock rom using `Androxyde's Flashtool <http://www.flashtool.net/downloads.php>`_. Last sure to be working version of Flashtool working with nicki was 0.9.18.6, newer versions could fail.
```

Comment:
_Last sure to be working version of Flashtool working with_
English module misbehaving